### ISSUE TITLE
Update accessibility for "Get Directions" link on VAMCs

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-team#25758]
## Issue description
The "Get Directions" link was flagged as a [508-defect-2 during the Vet Center staging review](https://github.com/department-of-veterans-affairs/va.gov-team/issues/25006). Similar functionality is seen on VAMC pages and should be remediated. 

## User story
As any user (but especially AT users), I want to know if a link will open a new tab and where it will link me to.
"Get directions" does not provide key information on _where_ directions will be procured from and that it opens a new tab. The latter can be particularly frustrating if unanticipated by assistive tech users.

## Details

<details>
<summary> Accessibility guidance </summary> 

- Limit the use of links or buttons that open new windows or tabs within Web content, since they can be disorienting for people, especially those who have difficulty perceiving visual content
- Name of the location is more screen reader specific for folks who navigate using the links list.  If every link says “Directions on Google maps“, it won’t be very useful. 
</details>

---

<details>
<summary> Main page </summary>

![image](https://user-images.githubusercontent.com/55411834/121061353-fb40c780-c780-11eb-9164-348527f392af.png)

</details>

---

<details>
<summary> Occurs repeatedly on the Locations page </summary>

![image](https://user-images.githubusercontent.com/55411834/121061470-1d3a4a00-c781-11eb-8207-804b4645cc48.png)

</details>

## Acceptance Criteria

- [ ] "Get directions" is labeled as "Directions on Google Maps" 
- [ ] Clicking on "Directions on Google Maps" does not open a new tab 
- [ ] Screen reader announcement of "Directions on Google Maps" is more specific, (i.e. includes Facility name)

**Comment by mmiddaugh** [comment]
Duplicate for Update Directions link behavior on VAMC pages #30917
